WalletWasabi/WabiSabi/Client/WabiSabiHttpApiClient.cs

```diff
@@ -65,6 +66,7 @@ private async Task<HttpResponseMessage> SendWithRetriesAsync(RemoteAction action
 		using CancellationTokenSource absoluteTimeoutCts = new(totalTimeout);
 		using CancellationTokenSource linkedCts = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, absoluteTimeoutCts.Token);
 		CancellationToken combinedToken = linkedCts.Token;
+		var httpClient = _httpClientFactory.CreateClient(_identity);
```

Comment:
I wonder whether it would be better to recreate the `httpClient` after a failure (so on retry) to re-trigger the expiration check. Not sure what is best

Reply:
There is no need to recreate the httpClient.